vpnglean/CMakeLists.txt

```diff
@@ -6,10 +6,46 @@ add_library(vpnglean STATIC IMPORTED GLOBAL)
 
 set_target_properties(vpnglean PROPERTIES FOLDER "Libs")
 
-if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+set(RUST_ARCH "")
```

Comment:
I am feeling a need to try and produce a rust helper function to produce linkable targets, like we do for golang to try and contain the complexity of cross compiling. But that's a task for a future PR.

Reply:
I am feeling the need to write a specification for target / architecture names so that everyone AGREESSSSSSS

Reply:
This is madness everyone just users whatever spelling / casing / words that come to mind

Reply:
Yeah, it's is a mess. I was beating my head against the same problem when trying to get apple universal builds working.